tmt/steps/execute/__init__.py

```diff
@@ -284,25 +297,35 @@ def check_beakerlib(self, test: "tmt.Test") -> List["tmt.Result"]:
             self.debug(
                 f"No result or state found in '{beakerlib_results_file}'.",
                 level=3)
-            data.note = 'beakerlib: Result/State missing'
-            return [tmt.Result(data=data, test=test)]
+            note = 'beakerlib: Result/State missing'
+            return [tmt.Result.from_test(
+                test=test,
+                result=actual_result,
+                note=note,
+                log=log,
+                duration=test.real_duration)]
 
         result = search_result.group(1)
         state = search_state.group(1)
 
         # Check if it was killed by timeout (set by tmt executor)
         if test.returncode == tmt.utils.PROCESS_TIMEOUT:
-            data.result = ResultOutcome.ERROR
-            data.note = 'timeout'
+            actual_result = ResultOutcome.ERROR
+            note = 'timeout'
             self.timeout_hint(test)
         # Test results should be in complete state
         elif state != 'complete':
-            data.result = ResultOutcome.ERROR
-            data.note = f"beakerlib: State '{state}'"
+            actual_result = ResultOutcome.ERROR
```

Comment:
Is this and `L313` assignment also needed? I see `actual_result = ResultOutcome.ERROR` on `L269`.

Reply:
Addressed in 7b113c75.

Reply:
Thanks.